actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -286,6 +282,8 @@ def image_tag(source, options = {})
       #   image_alt('underscored_file_name.png')
       #   # => Underscored file name
       def image_alt(src)
+        ActiveSupport::Deprecation.warn("ActionView::Helpers::AssesTagHelper.image_alt is deprecated and will be removed from Rails 5.2. You must exlicitly set alt text on images.")
```

Comment:
Don't need to talk about the module name too. Only `image_alt is deprecated` is good. And it is going to be removed from Rails 6.0.